### PR TITLE
Adds in metadata ingestion for ZPLSG

### PR DESCRIPTION
**Pull request by cdobs**

### CODE REVIEW

**Comment by cdobs** [comment]
Updates the ZPLSG (global) ingestions to use the new metadata parser. 

Redmine ticket https://redmine.oceanobservatories.org/issues/15166 shows the name of the new parser and an ingestion example. In this example Mark ingested these metadata as "recovered_host" but I confirmed with him that "recovered_inst" works as well and we prefer to use that. I used "recovered_inst" in my test ingestion on dev. 

We decided to ingest the metadata from the averaged nc files so I updated the filemasks to point to those.

File: GI02HYPM/GI02HYPM_R00007_ingest.csv [hunk 1]
```diff
@@ -1,6 +1,6 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/ZPLSG_sn55109/DATA/20*/*.01A,GI02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Pending,parser not available
-#,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/ZPLSG_sn55111/DATA/20*/*.01A,GI02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Pending,parser not available
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/ZPLSG_sn55109/processed/*/*Averaged.nc,GI02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Available,
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GI02HYPM/R00007/instruments/ZPLSG_sn55111/processed/*/*Averaged.nc,GI02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GI02HYPM/GI02HYPM_R00007_ingest.csv:3]: looks good

File: GI02HYPM/GI02HYPM_R00008_ingest.csv [hunk 2]
```diff
@@ -1,6 +1,6 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,/omc_data/whoi/OMC/GI02HYPM/R00008/instruments/ZPLSG_sn55106/DATA/20*/*.01A,GI02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Pending,parser not available
-#,/omc_data/whoi/OMC/GI02HYPM/R00008/instruments/ZPLSG_sn55108/DATA/20*/*.01A,GI02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Pending,parser not available
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GI02HYPM/R00008/instruments/ZPLSG_sn55106/processed/*/*Averaged.nc,GI02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Available,
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GI02HYPM/R00008/instruments/ZPLSG_sn55108/processed/*/*Averaged.nc,GI02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GI02HYPM/GI02HYPM_R00008_ingest.csv:3]: looks good

File: GP02HYPM/GP02HYPM_R00004_ingest.csv [hunk 3]
```diff
@@ -1,6 +1,6 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,/omc_data/whoi/OMC/GP02HYPM/R00004/instruments/ZLPSG_sn55093/DATA/20*/,GP02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Not Expected,not recovered
-#,/omc_data/whoi/OMC/GP02HYPM/R00004/instruments/ZLPSG_sn55092/DATA/20*/,GP02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Not Expected,not recovered
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GP02HYPM/R00004/instruments/ZPLSG_sn55093/processed/*/*Averaged.nc,GP02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Available,
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GP02HYPM/R00004/instruments/ZPLSG_sn55092/processed/*/*Averaged.nc,GP02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GP02HYPM/GP02HYPM_R00004_ingest.csv:3]: looks good

File: GP02HYPM/GP02HYPM_R00005_ingest.csv [hunk 4]
```diff
@@ -1,6 +1,6 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,/omc_data/whoi/OMC/GP02HYPM/R00005/instruments/ZLPSG_sn55110/DATA/20*/*.01A,GP02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Pending,no parser
-#,/omc_data/whoi/OMC/GP02HYPM/R00005/instruments/ZLPSG_sn55104/DATA/20*/*.01A,GP02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Pending,no parser
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GP02HYPM/R00005/instruments/ZPLSG_sn55110/processed/*/*Averaged.nc,GP02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Available,
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GP02HYPM/R00005/instruments/ZPLSG_sn55104/processed/*/*Averaged.nc,GP02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GP02HYPM/GP02HYPM_R00005_ingest.csv:3]: looks good

File: GP02HYPM/GP02HYPM_R00006_ingest.csv [hunk 5]
```diff
@@ -1,6 +1,6 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/ZPLSG_sn55093/DATA/20*/*.01A,GP02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Pending,no parser
-#,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/ZPLSG_sn55092/DATA/20*/*.01A,GP02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Pending,no parser
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/ZPLSG_sn55093/processed/*/*Averaged.nc,GP02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Available,
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GP02HYPM/R00006/instruments/ZPLSG_sn55092/processed/*/*Averaged.nc,GP02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GP02HYPM/GP02HYPM_R00006_ingest.csv:3]: looks good

File: GP02HYPM/GP02HYPM_R00007_ingest.csv [hunk 6]
```diff
@@ -1,6 +1,6 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/ZPLSG_sn55068/DATA/20*/*.01A,GP02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Pending,no parser
-#,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/ZPLSG_sn55107/DATA/20*/*.01A,GP02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Pending,no parser
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/ZPLSG_sn55068/processed/*/*Averaged.nc,GP02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Available,
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GP02HYPM/R00007/instruments/ZPLSG_sn55107/processed/*/*Averaged.nc,GP02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GP02HYPM/GP02HYPM_R00007_ingest.csv:3]: looks good

File: GP02HYPM/GP02HYPM_R00008_ingest.csv [hunk 7]
```diff
@@ -1,6 +1,6 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,/omc_data/whoi/OMC/GP02HYPM/R00008/instruments/ZPLSG_sn55092/DATA/20*/*.01A,GP02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Pending,no parser
-#,/omc_data/whoi/OMC/GP02HYPM/R00008/instruments/ZPLSG_sn55093/DATA/20*/*.01A,GP02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Pending,no parser
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GP02HYPM/R00008/instruments/ZPLSG_sn55092/processed/*/*Averaged.nc,GP02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Available,
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GP02HYPM/R00008/instruments/ZPLSG_sn55093/processed/*/*Averaged.nc,GP02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GP02HYPM/GP02HYPM_R00008_ingest.csv:3]: looks good

File: GP02HYPM/GP02HYPM_R00009_ingest.csv [hunk 8]
```diff
@@ -1,6 +1,6 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,/omc_data/whoi/OMC/GP02HYPM/R00009/instruments/ZPLSG_sn55068/DATA/20*/*.01A,GP02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Pending,no parser
-#,/omc_data/whoi/OMC/GP02HYPM/R00009/instruments/ZPLSG_sn55107/DATA/20*/*.01A,GP02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Pending,no parser
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GP02HYPM/R00009/instruments/ZPLSG_sn55068/processed/*/*Averaged.nc,GP02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Available,
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GP02HYPM/R00009/instruments/ZPLSG_sn55107/processed/*/*Averaged.nc,GP02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GP02HYPM/GP02HYPM_R00009_ingest.csv:3]: looks good

File: GS02HYPM/GS02HYPM_R00003_ingest.csv [hunk 9]
```diff
@@ -1,6 +1,6 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,/omc_data/whoi/OMC/GS02HYPM/R00003/instruments/ZPLSG_sn55105/DATA/201*/*.01A,GS02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Pending,no parser
-#,/omc_data/whoi/OMC/GS02HYPM/R00003/instruments/ZPLSG_sn55106/DATA/201*/*.01A,GS02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Pending,no parser
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GS02HYPM/R00003/instruments/ZPLSG_sn55105/processed/*/*Averaged.nc,GS02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Available,
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GS02HYPM/R00003/instruments/ZPLSG_sn55106/processed/*/*Averaged.nc,GS02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Available,
```
Comment (rtravis-whoi) [inline GS02HYPM/GS02HYPM_R00003_ingest.csv:3]: looks good

**Review by rtravis-whoi — APPROVED** [review]

File: GI02HYPM/GI02HYPM_R00006_ingest.csv [hunk 10]
```diff
@@ -1,6 +1,6 @@
 parser,filename_mask,reference_designator,data_source,status,notes
-#,/omc_data/whoi/OMC/GI02HYPM/R00006/instruments/ZPLSG_sn55069/DATA/20*/*.01A,GI02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Pending,parser not available
-#,/omc_data/whoi/OMC/GI02HYPM/R00006/instruments/ZPLSG_sn55066/DATA/20*/*.01A,GI02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Pending,parser not available
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GI02HYPM/R00006/instruments/ZPLSG_sn55069/processed/*/*Averaged.nc,GI02HYPM-MPM01-02-ZPLSGA009,recovered_inst,Available,
+mi.dataset.driver.zplsc_c.zplsc_echogram_uploader,/omc_data/whoi/OMC/GI02HYPM/R00006/instruments/ZPLSG_sn55066/processed/*/*Averaged.nc,GI02HYPM-MPM01-02-ZPLSGA010,recovered_inst,Available,
```
Comment (cooleyky) [inline GI02HYPM/GI02HYPM_R00006_ingest.csv:3]: looks good

File: GI02HYPM/GI02HYPM_R00007_ingest.csv (hunk 1 above)
Comment (cooleyky) [inline GI02HYPM/GI02HYPM_R00007_ingest.csv:3]: looks good

File: GI02HYPM/GI02HYPM_R00008_ingest.csv (hunk 2 above)
Comment (cooleyky) [inline GI02HYPM/GI02HYPM_R00008_ingest.csv:3]: looks good

File: GP02HYPM/GP02HYPM_R00004_ingest.csv (hunk 3 above)
Comment (cooleyky) [inline GP02HYPM/GP02HYPM_R00004_ingest.csv:3]: looks good

File: GP02HYPM/GP02HYPM_R00005_ingest.csv (hunk 4 above)
Comment (cooleyky) [inline GP02HYPM/GP02HYPM_R00005_ingest.csv:3]: looks good

File: GP02HYPM/GP02HYPM_R00006_ingest.csv (hunk 5 above)
Comment (cooleyky) [inline GP02HYPM/GP02HYPM_R00006_ingest.csv:3]: looks good

File: GP02HYPM/GP02HYPM_R00008_ingest.csv (hunk 7 above)
Comment (cooleyky) [inline GP02HYPM/GP02HYPM_R00008_ingest.csv:3]: looks good

**Review by cooleyky — COMMENTED** [review]
looks good

**Review by cooleyky — APPROVED** [review]
looks good

**Pull request merged**